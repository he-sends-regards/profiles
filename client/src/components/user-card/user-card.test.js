import React from 'react';
import renderer from 'react-test-renderer';
import UserCard from './user-сard.jsx';

const user = {
  _id: '1',
  name: 'Dan',
  email: 'test@gmail.com',
  password: '123456',
  isAdmin: true,
};

const noop = () => {};

it('Renders correctly', () => {
  const tree = renderer
      .create(
          <UserCard
            user={user}
            setIsUserDataChanged={noop}
          />,
      )
      .toJSON();
  expect(tree).toMatchSnapshot();
});
