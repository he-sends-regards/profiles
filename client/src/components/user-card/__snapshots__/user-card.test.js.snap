// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className="user-card card"
>
  <div
    className="user-card_body card-body"
  >
    <div
      className="card-title h5"
    >
      Dan
    </div>
    <div
      className="mb-2 text-muted card-subtitle h6"
    >
      test@gmail.com
    </div>
    <p
      className="d-flex flex-column card-text"
    >
      <span>
        Is admin: 
        true
      </span>
    </p>
    <div>
      <a
        className="card-link"
        onClick={[Function]}
        style={
          Object {
            "color": "red",
            "cursor": "pointer",
          }
        }
      >
        Delete
      </a>
    </div>
  </div>
</div>
`;
